app/assets/stylesheets/preact/onboarding-modal.scss

```diff
@@ -360,6 +360,34 @@ $onboarding-user-selected-hover: rgba(71, 85, 235, 0.2);
     border: 1px solid var(--base-10);
   }
 
+  .current-user-info {
+    text-align: center;
+    margin-bottom: 1rem;
+
+    .current-user-avatar-container {
+      height: 80px;
+    }
+
+    .current-user-avatar {
+      border-radius: 100%;
+      height: inherit;
+      object-fit: cover;
+      border: 2px solid var(--base-90);
+    }
+
+    h3 {
+      margin-top: 6px;
+    }
+
+    p {
+      font-size: 16px;
```

Comment:
at the risk of having different font sizes...do we have a global `p` style we can use here? i wonder if crayons has set/reusable font sizes that we can use here

Reply:
`app/assets/stylesheets/config/_import.scss` is the file containing information about the various crayons variables, but to use them, we need to make sure we're importing that file here.

If we wanted the equivalent of `16px`, then we can use `$fs-base`, which is defined as `1 rem`.

Reply:
Updated!